set.mm

```diff
@@ -645047,6 +645080,263 @@ fixed reference functional determined by this vector (corresponding to
+      cn crab eleq2i adantl fveqeq2 elrab sylib elpwid sseld 3impa elfznn nnred
+      cpw imp 3expa ex ssrdv ltso mpi fzfid ssfid fz1iso syl2anc wbr wi cab cvv
+      soss wf1o wb df-isom 3ad2ant3 simprd f1oeq2d biimpd 3adant3 mpd f1of ffnd
+      oveq2 ovexd fnexd fss biimp a1i ralimdva adantr oveq2d raleqdv raleqbidva
+      mpbid jca feq1 fveq1 breq12d imbi2d 2ralbidv anbi12d elabd crn cmpt simpr
+      sylibr rneqd rnexg fvmptd 3ad2ant1 wfn dff1o2 simp3d eqtrd eqcomd eximdv
+      df-rex ralrimiva dffo3 mpbird ) AEFHUDZEFHULZUAUMZUBUMZHUEZUFZUBEUGZUAFQZ
+      RZAUUGUUMAEFHUHZUUGABCDEFGHIJKLMNOPUIUUOUUGHUJUKZUUOUUGUUPREFHUNUOUPUQAUU
+      LUAFAUUHFSZRZUUIESZUUKRZUBURZUULUURUSUUHUTUEZVAVBZUUHTTUUIVCZUBURZUVAUURU
+      UHTVDZUUHVESUVEUURUUHVFVGZUVFUURUCUUHVFUURUCUMZUUHSZUVHVFSZAUUQUVIUVJAUUQ
+      UVIVHZUVHUVKUVHUSJVAVBZSZUVHVISAUUQUVIUVMUURUVIUVMUURUUHUVLUVHUURUUHUVLUU
+      RUUHUVLWAZSZUVBIUFZUURUUHKUMZUTUEIUFZKUVNVJZSZUVOUVPRUUQUVTAUUQUVTFUVSUUH
+      NVKUOVLUVRUVPKUUHUVNUVQUUHIUTVMVNVOZUPVPZVQWBVRUVHJVSUQVTWCWDWEUVGVFTVDUV
+      FWFUUHVFTWPWGUQUURUVLUUHUURUSJWHUWBWIUUHTUBWJWKUURUVDUUTUBUURUVDUUTAUUQUV
+      DUUTAUUQUVDVHZUUSUUKUWCUUIUSIVAVBZUVLGUMZULZBUMZCUMZTWLZUWGUWEUEZUWHUWEUE
+      ZTWLZWMZCUWDQBUWDQZRZGWNZSUUSUWCUWOUWDUVLUUIULZUWIUWGUUIUEZUWHUUIUEZTWLZW
+      MZCUWDQZBUWDQZRGUUIWOUWCUWDUUIWOUWCUWDUUHUUIUWCUWDUUHUUIWQZUWDUUHUUIULZUW
+      CUVCUUHUUIWQZUXDUWCUXFUWIUWTWRZCUVCQZBUVCQZUVDAUXFUXIRZUUQUVDUXJBCUVCUUHT
+      TUUIWSUOWTZUPAUUQUXFUXDWMUVDUURUXFUXDUURUVPUXFUXDWRUURUVOUVPUWAXAZUVPUVCU
+      WDUUHUUIUVBIUSVAXHXBUQXCXDXEZUWDUUHUUIXFUQZXGUWCUSIVAXIXJUWCUWQUXCUWCUXEU
+      UHUVLVGZUWQUXNAUUQUXOUVDUWBXDUWDUUHUVLUUIXKWKUWCUXACUVCQZBUVCQZUXCUWCUXIU
+      XQUWCUXFUXIUXKXAUWCUXHUXPBUVCUWCUWGUVCSZRZUXGUXACUVCUXGUXAWMUXSUWHUVCSRUW
+      IUWTXLXMXNXNXEUWCUXPUXBBUVCUWDUWCUVBIUSVAAUUQUVDUVPUURUVPUVDUXLXOVRXPZUWC
+      UXPUXBWRUXRUWCUXACUVCUWDUXTXQXOXRXSXTUWEUUIUFZUWFUWQUWNUXCUWDUVLUWEUUIYAU
+      YAUWMUXABCUWDUWDUYAUWLUWTUWIUYAUWJUWRUWKUWSTUWGUWEUUIYBUWHUWEUUIYBYCYDYEY
+      FYGEUWPUUIOVKYKZUWCUUJUUHUWCUUJUUIYHZUUHUWCUUSUUJUYCUFZUYBAUUQUUSUYDWMUVD
+      AUUSUYDAUUSRZDUUIDUMZYHZUYCEHWOHDEUYGYIUFUYEPXMUYEUYFUUIUFZRUYFUUIUYEUYHY
+      JYLAUUSYJZUYEUUSUYCWOSUYIUUIEYMUQYNWDYOXEUWCUXDUYCUUHUFZUXMUXDUUIUWDYPZUU
+      IUJUKZUYJUXDUYKUYLUYJVHUWDUUHUUIYQUOYRUQYSYTXTWCWDUUAXEUUKUBEUUBYKUUCXTUU
+      FUUNWRAUBUAEFHUUDXMUUE $.
+  $}
+
+  ${
+    $d A a p $.  $d A f p $.  $d A g p $.  $d B g p $.  $d B p x y $.
+    $d K a x y $.  $d K f x y $.  $d N a $.  $d N f $.  $d a p ph x y $.
+    $d f p ph x y $.  $d g p ph $.
+    sticksstones4.1 $e |- ( ph -> N e. NN ) $.
+    sticksstones4.2 $e |- ( ph -> K e. NN0 ) $.
+    sticksstones4.3 $e |- B = { a e. ~P ( 1 ... N ) | ( # ` a ) = K } $.
+    sticksstones4.4 $e |- A = { f | ( f : ( 1 ... K ) --> ( 1 ... N ) /\ A. x
+    e. ( 1 ... K ) A. y e. ( 1 ... K )
+    ( x < y -> ( f ` x ) < ( f ` y ) ) ) } $.
+    $( Equinumerosity lemma for sticks and stones .)  (Contributed by metakunt,
```

Comment:
```suggestion
    $( Equinumerosity lemma for sticks and stones.  (Contributed by metakunt,
```

this is probably still a draft, but I'll note this typo

Reply:
Should be fixed now

Reply:
Actually the theorem I've wanted to prove is proven, it can be reviewed now